src/sql/src/kafka_util.rs

```diff
@@ -516,8 +518,13 @@ pub fn generate_ccsr_client_config(
     }
 
     let mut ccsr_options = extract(
-        &mut ccsr_options,
-        &[Config::string("username"), Config::string("password")],
+        ccsr_options,
+        &[
+            Config::string("username"),
+            Config::string("password"),
+            // An old migration added this field in some avro cases, so we remove it here
```

Comment:
oof.

Reply:
actually, this comment might be wrong, but I couldn't figure out the true reason

Reply:
We should definitely figure out the real reason before merging this. I'm happy to help. What's the test that fails without this?